### DISCUSSION TITLE
Install Elementary OS with Windows 11

### DISCUSSION THREAD

**Discussion by ryse-amp** [elementary/triage#582] [Q&A]
I recently recover my notebook with "Windows 11 Home simple lenguage (22H2)" installed. Now i would like to try
with elementary OS 7.1 Horus dual mode. The features of my system:

   

Processor: 11Th Gen Intel(R) Core(TM) i7-1165G7 @ 2.80GHz 
Ram: 16GB
video: Nvidia Gforce MX450

1.¿How can i know if it is neccesary to create the three different disk spaces, i mean, swap,home and ---¿mount?---?
?. ¿what cuota should be assign to each 'disk spaces'? Because, i saw two different old videos in youtube when one create the three spaces and the other one did not.

2.¿How about los drivers of the video card, is there the corrects ones in elementary OS?()

Regards

**Comment by sudanchapagain** [comment]
I too dual boot, i have four partitions but and i would recommend just three. A `root`, `home`, & `EFI`.

### Why no swap?

see: <https://askubuntu.com/questions/1234838/is-it-necessary-to-have-a-home-and-swap-partitions-in-20-04>

### Why seperate home & root?

ElementaryOS upgrades cannot be installed with reinstalling the whole os, so when ElementaryOS 8 comes out i want to be able to easily upgrade by seperating the programs (root) and user data (home).

### My partition layout

1. Home
    **size**: 50GB (this is your choice, however much you want to allocate)
    **format**: BTRFS
2. Root
    **size**: 50GB (someone on discord said around 30GB, ideally 50GB)
    **format**: BTRFS
3. Swap
    **size**: 16GB (i have 16GB RAM so 16GB)
4. EFI (a new EFI because i had heard windows might sometimes mess  the default EFI that windows made. So just to be safe i made a new EFI of 512MB)

### Drivers

When installing click on proprietary and extra drivers options. It should install everything by default.
